cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -107,7 +107,7 @@ function create_resource_from_string() {
   local -r config_name=$4;
   local -r namespace=$5;
   while [ ${tries} -gt 0 ]; do
-    echo "${config_string}" | ${KUBECTL} ${KUBECTL_OPTS} apply -f - && \
+    echo "${config_string}" | ${KUBECTL} ${KUBECTL_OPTS} --namespace="${namespace}" apply -f - && \
```

Comment:
NOTE: Added back namespace to this function as it is required per the code comment:
https://github.com/kubernetes/kubernetes/blob/ec7f7a5c5fa7c7751e476a697e79b1527f8ac8b3/cluster/addons/addon-manager/kube-addons.sh#L188-L194

This will not affect the apply/ensure loop, which is handled by another functions (`nsure_addons`, `reconcile_addons`).